### ISSUE TITLE
Skeleton loading: Design system website contextual example

### ISSUE THREAD

**Issue by Spark450** [GovAlta/ui-components#2119]
## Acceptance Criteria:

- Add the following links to the "TOC" component:
  - Loading a table skeleton
  - Load a full page skeleton
- Include 2 examples shown in [figma design](https://www.figma.com/design/aeIJGExcpx9ecqnIVwImi1/Component-Pages?node-id=87-340577&t=wm8tst8hY9zJU4Cd-1)  
  - Loading a table skeleton
  - Load a full page skeleton

**Comment by mxsoco** [comment]
@Spark450 There's a skeleton loading example on the Basic Layout page: https://design.alberta.ca/patterns/layout#tab-0. Is the "Load a full page skeleton" still something that should be added to the component page?

**Comment by Spark450** [comment]
@mxsoco Yes, I think in this case we would still want a full page example on the component page as outlined in the Figma design.

**Comment by Spark450** [comment]
@mxsoco Thanks for confirming. On reviewing again, there is limited additional value in more examples here. I will close this issue.
